src/device-registry/utils/translate.js

```diff
@@ -18,6 +18,15 @@ const translateUtil = {
       for (const healthTip of healthTips) {
         const translatedTip = { ...healthTip };
         const translations = await tipsTranslations(index, targetLanguage);
+
+        if (!translations.title || !translations.description) {
```

Comment:
Thanks @BenjaminSsempala , just left a followup Slack DM accordingly. Sincere apologies for dragging review thread.